puppet/zulip_ops/manifests/profile/zmirror_personals.pp

```diff
@@ -69,6 +67,4 @@
   # Building patched libzephyr4-krb5 from davidben's roost branch and installing that
   #  (https://github.com/davidben/zephyr/commit/252258d38ebd0e79b261db336c1f74f261b77128)
   #  (to add ZLoadSession/ZDumpSession).
```

Comment:
This patch was upstreamed in Zephyr 3.1-rc0 (zephyr-im/zephyr@e6a545e75963456834f120cfc62983eb4d001490), so we should be able to return to stock 3.1.2.

Reply:
Updated.